include/rocksdb/db.h

```diff
@@ -276,6 +276,16 @@ class DB {
       const std::string& input, std::string* output,
       const CompactionServiceOptionsOverride& override_options);
 
+  // Create a new DB by merging multiple disjoint DBs.
+  // Memtable merge is not supported. All source DBs must be flushed
+  // beforehand. If any of the source DBs is written during the merge process,
+  // the operation will be aborted.
+  static Status CreateFromDisjointInstances(
+      const DBOptions& db_options, const std::string& name,
+      const std::vector<ColumnFamilyDescriptor>& column_families,
```

Comment:
How should `column_families` be specified? For example, merging two db with multipe cfs?

Reply:
The same way as opening a normal DB. It must be specified the same way (order, name) as existing instances, so that column family id are identical.

Reply:
What if the ID doesn't match? For example, in the old days, we don't have multiple column families. And in the some time of history, we don't have some extra but deleted column families.

Reply:
Updated to use string match instead.